amr-wind/wind_energy/ShearStress.H

```diff
@@ -216,6 +216,9 @@ struct ShearStressDonelan
         case amr_wind::MOData::ThetaCalcType::SURFACE_TEMPERATURE:
             flux = 0.0012 * wspd_mean * (theta_surface - theta_mean);
             break;
+        case amr_wind::MOData::ThetaCalcType::NEAR_SURFACE_TEMPERATURE:
+            amrex::Print() << "To be implemented..." << std::endl;
```

Comment:
this print statement is causing the GPU-related checks to fail. No worries for now!